server/tools/attestation-service-ctl/src/attestation.rs

```diff
@@ -0,0 +1,24 @@
+use anyhow::{anyhow, Result};
+use std::fs;
+use std::path::Path;
+
+use crate::attestation_api::attestation_service_client::AttestationServiceClient;
+use crate::attestation_api::{AttestationRequest, AttestationResponse};
+
+pub async fn attestation_cmd(evidence_path: &Path, socket: &str) -> Result<()> {
+    let evidence =
+        fs::read_to_string(evidence_path).map_err(|e| anyhow!("Read evidence error: {:?}", e))?;
+
+    let request = AttestationRequest {
+        evidence: evidence.into_bytes(),
+        user: None,
+    };
+
+    let mut client = AttestationServiceClient::connect(format!("http://{}", socket)).await?;
+    let response: AttestationResponse = client.attestation(request).await?.into_inner();
+    let results = String::from_utf8(response.attestation_results)?;
+    debug!("Attestation Results:\n");
+    info!("{}", results);
```

Comment:
Can you combine these into a single call maybe (multi-line log output is confusing).